src/main/java/racing/InputView.java

```diff
@@ -0,0 +1,16 @@
+package racing;
+
+import java.util.Scanner;
+
+public class InputView {
+    public static Scanner scanner = new Scanner(System.in);
+    public static int inputCarCount() {
```

Comment:
- 변수와 메서드의 구분을 위해 개행을 추가허여 가독성을 높여보면 좋을 것 같습니다.
```suggestion
    public static Scanner scanner = new Scanner(System.in);
    
    public static int inputCarCount() {
```